app/assets/stylesheets/equalizer.scss

```diff
@@ -0,0 +1,81 @@
+* {
```

Comment:
Why is this file called equalizer instead of common, main, base, or something like that?